.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Create a bug report for rustfmt
```

Comment:
I wonder whether the average user would know which issue template to use :thinking: What if we made this a little more explicit like

```suggestion
about: Report rustfmt errors, failing to format inputs, etc.
```

Reply:
Yeah, I agree. Thanks for the suggestion!